images/image.go

```diff
@@ -187,13 +187,13 @@ func Manifest(ctx context.Context, provider content.Provider, image ocispec.Desc
 			return descs, nil
 
 		}
-		return nil, errors.Wrap(errdefs.ErrNotFound, "could not resolve manifest")
+		return nil, errors.Wrapf(errdefs.ErrNotFound, "could not resolve manifest %v", desc.Digest)
```

Comment:
Isn't this case something like `"unexpected media type %s for %s", desc.MediaType, desc.Digest` ?

I'm not sure why this is "not found"

Reply:
The class of error is that the manifest is not found. It could not be found because it was unresolvable.

Reply:
Not found is the correct class, but we could add the media type here to make it clearer here why the resolution couldn't happen

Reply:
Ok, take care of this in a follow up.

Reply:
See https://github.com/containerd/containerd/pull/1861